### DISCUSSION TITLE
Rotate file without quality issue

### DISCUSSION THREAD

**Discussion by HypeillingerChillinger** [SixLabors/ImageSharp#2166] [Ideas]
This is actually a question but I did not find the correct way to create one.

What I do is rotating the image like this:

```c#
image.Mutate(x => x.Rotate(rotateMode));
```
which is exactly what I need - but if saved it applies the quality (JPEG) once again. 

What I tried is
```c#
image.SaveAsync(newFullFilePath, new JpegEncoder());
// or
image.SaveAsJpegAsync(newFullFilePath);
```
The quality seems to decrease on each iteration.

**Comment by dlemstra** [comment]
Here is a great article that explains why this happens: https://cloudinary.com/blog/why_jpeg_is_like_a_photocopier.

**Comment by JimBobSquarePants** [comment]
> This is actually a question but I did not find the correct way to create one.

I would suggest starting at the readme

**Comment by HypeillingerChillinger** [comment]
Actually I edited my question before you read it. My fault. That the quality of a JPG decreases in each iteration is clear. I wanted to find a way how to change the orientation in the EXIF Metadata in order to rotate without changing the image data and therefore without loss of quality.

**Reply by br3aker** [reply]
Metadata editing without re-encoding is not supported *yet*.

**Comment by JimBobSquarePants** [comment]
Locking because this isn’t a QA channel.
